### PR TITLE
Front-end changes to the UI

### PR DESCRIPTION
**Pull request by brett-anderson**
Added Bootstrap, and preliminary styles to the front page.

Added css, and changed structure of the Idea view.

### CODE REVIEW

File: config/environments/development.rb [hunk 1]
```diff
@@ -37,4 +37,5 @@
 
   # To Mailer get the correct url
   config.action_mailer.default_url_options = { :host => 'localhost:3000' }
+  config.assets.initialize_on_precompile = true
```
Comment (swalberg) [inline config/environments/development.rb:40]: Was this needed to get anything to work? I think this is the default for the development environment.

File: app/views/ideas/index.html.slim [hunk 2]
```diff
@@ -1,24 +1,27 @@
-h1 Ideas
+.container
```
Comment (swalberg) [inline app/views/ideas/index.html.slim:1]: The layout already wraps the view in a `.container`, is it required again?

**Comment by dbernar1** [comment]
Hey Brett,

Can we get a screenshot perhaps?

**Comment by amirci** [comment]
The easiest way to get a screenshot is to download the pull request code and look at it.

**Pull request closed without merging**